python/lsst/ts/envsensors/mock/mock_temperature_sensor.py

```diff
@@ -15,17 +15,17 @@
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 __all__ = ["MockTemperatureSensor"]
 
-import asyncio
 import logging
 import random
 import time
 
-import numpy as np
-
-from lsst.ts.ess.sensors.sel_temperature_reader import DELIMITER
+from lsst.ts.envsensors.sel_temperature_reader import DELIMITER
```

Comment:
In library code I gently suggest using relative imports, because it reduces the risk of circular imports.

Reply:
Good suggestion, thanks. Done.